apps/etterlatte-behandling/src/main/kotlin/oppgave/GosysOppgaveKlient.kt

```diff
@@ -30,8 +31,30 @@ data class GosysApiOppgave(
     val fristFerdigstillelse: LocalDate
 )
 
+data class GosysEndreSaksbehandlerRequest(
+    val versjon: Long,
+    val tilordnetRessurs: String
```

Comment:
Skal vi mappe om denne til å kalles saksbehandler slik at den er lik for begge oppgave objektene?

Reply:
Feltet har det navnet i oppgave-api, og typen brukes direkte i serialiseringen til json. Eller misforstår jeg hva du tenker på..?

Reply:
Jeg blingsa litt på bruken her. Det er fint som det er!